integration/security_ssh_key/security_ssh_key_test.go

```diff
@@ -0,0 +1,65 @@
+package security_ssh_key_test
+
+import (
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	testhelpers "github.com/maximilien/softlayer-go/test_helpers"
+)
+
+var _ = Describe("SoftLayer Security SSH keys", func() {
+	var (
+		err             error
+		securityService softlayer.SoftLayer_Security_Ssh_Key_Service
+	)
+
+	BeforeEach(func() {
+		securityService, err = testhelpers.CreateSecuritySshKeyService()
+		Expect(err).ToNot(HaveOccurred())
+
+		testhelpers.TIMEOUT = 30 * time.Second
+		testhelpers.POLLING_INTERVAL = 10 * time.Second
+	})
+
+	Context("SoftLayer_Security_Ssh_Key", func() {
+		It("creates an SSH key, update it, and delete it", func() {
```

Comment:
@posijon and I are working to remove the `SL_GO_TEST_SSH_KEY_PATH1` and ...2 dependencies. So this part will likely need to change to calling a `testhelpers` function